cli/cdi/__init__.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import pathlib
+from importlib import metadata
+
+try:
+    __version__ = metadata.version("mbse_docker_images")
+except metadata.PackageNotFoundError:  # pragma: no cover
+    __version__ = "0.0.0+unknown"
```

Comment:
Package name mismatch with `pyproject.toml`, where it's called "cdi". This causes `cdi --version` to be broken in both `uv tool` and "normal" editable installs (<https://github.com/DSD-DBS/capella-dockerimages/pull/392#discussion_r2044076398>).

```suggestion
try:
    __version__ = metadata.version("cdi")
except metadata.PackageNotFoundError:  # pragma: no cover
    __version__ = "0.0.0+unknown"
```

Reply:
Thanks, good catch!